doc/security.md

```diff
@@ -10,3 +10,12 @@ In the rare event that you find a vulnerability in pub.dev, contact us at
 
 For more information about how Google handles security issues, see
 [Google’s security philosophy](https://www.google.com/about/appsecurity/).
+
+## Security advisories
+
+In case of a vulnarability regarding a specific package, use GitHub’s [security
```

Comment:
Security advisories for pub packages can be published through the Github advisory database. Advisories from the Github advisory database will be surfaced in the `pub` client.